modules/gcp-gke/cluster.tf

```diff
@@ -8,9 +8,8 @@ terraform {
     }
 
     google-beta = {
```

Comment:
Please move this to `terraform.tf` as per the style guide, doesn't make sense to be hidden in one of the module resource file.

Reply:
ah sorry I was blindly following https://github.com/honestbank/terraform-gcp-sql/tree/main/modules/google_service_networking_connection